example.js

```diff
@@ -5,6 +5,14 @@ const fastify = require('fastify')()
 fastify
   .register(require('./index'))
   .after(() => {
+    // Add this to serve the _next assets
+    fastify.next('/_next/*', (app, req, reply) => {
+      return app.handleRequest(req.req, reply.res)
+        .then(() => {
+          reply.sent = true
+        })
+    })
```

Comment:
this block does not really look nice. Is it needed? should we just include this in the module?

Reply:
You right it was ugly and unnecessary.
Implemented the static asset handling in the plugin, so I removed the whole example code.
Some side note / question:
- Added an `onClose` hook for closing the next app.
- Also added tests for the implementation, but please note, that we currently testing the next app build in `NODE_ENV=production` only. If you try to run the tests in other NODE_ENV, then it will fail, because the asset map is different dependending on the environment.
- Should we test the asset handling in develoment environment too? If I not mistaken, the he hard part is that we build the next app before the tests, so if we want to add tests in other environment, than we need to add a different npm script eg `test:dev` and a different test file eg `test-development.js`